jest.config.js

```diff
@@ -2,5 +2,18 @@ module.exports = {
   displayName: "riipen-ui",
   name: "riipen-ui",
   roots: ["<rootDir>/packages/riipen-ui"],
-  setupFilesAfterEnv: ["<rootDir>/packages/riipen-ui/test/setup.js"]
+  setupFilesAfterEnv: ["<rootDir>/packages/riipen-ui/test/setup.js"],
+  testRegex: "/*.test.jsx$",
+  collectCoverage: true,
+  coverageReporters: ["lcov"],
+  coverageDirectory: "test-coverage",
+  coverageThreshold: {
+   global: {
+   branches: 0,
+   functions: 0,
+   lines: 0,
+   statements: 0
+   }
+  },
+  moduleDirectories: ["node_modules", "src"]
```

Comment:
@imogenem did you mean for all of this to be in the current PR?

Reply:
@jordanell No I didn't, it's removed now